flang/test/Semantics/OpenACC/bug135810-2.f90

```diff
@@ -0,0 +1,12 @@
+! RUN: %python %S/../test_errors.py %s %flang -fopenacc
+integer function square(x)
+    implicit none
+    integer, intent(in) :: x
+!ERROR: logical expression or object list expected
+    !$acc parallel self(,)
+    !$acc end parallel
+!ERROR: logical expression or object list expected
```

Comment:
Do you have a test that `!$acc parallel self(object-list)` would cause a diagnostic? (as `self` only takes an object list when used on an `update` construct).